administrator/components/com_content/src/Model/ArticlesModel.php

```diff
@@ -302,7 +302,8 @@ protected function getListQuery()
 			->join('LEFT', $db->quoteName('#__users', 'ua'), $db->quoteName('ua.id') . ' = ' . $db->quoteName('a.created_by'))
 			->join('INNER', $db->quoteName('#__workflow_associations', 'wa'), $db->quoteName('wa.item_id') . ' = ' . $db->quoteName('a.id'))
 			->join('INNER', $db->quoteName('#__workflow_stages', 'ws'), $db->quoteName('ws.id') . ' = ' . $db->quoteName('wa.stage_id'))
-			->join('INNER', $db->quoteName('#__workflows', 'w'), $db->quoteName('w.id') . ' = ' . $db->quoteName('ws.workflow_id'));
+			->join('INNER', $db->quoteName('#__workflows', 'w'), $db->quoteName('w.id') . ' = ' . $db->quoteName('ws.workflow_id'))
+			->where($db->quoteName('wa.extension') . ' = ' . $db->quote('com_content.article');
```

Comment:
```suggestion
			->where($db->quoteName('wa.extension') . ' = ' . $db->quote('com_content.article'));
```
Have we forgotten this one?